microlith/grafana/provisioning/dashboards/single-cluster-overview.json

```diff
@@ -428,6 +428,16 @@
               {
                 "id": "custom.width",
                 "value": 100
+              },
+              {
+                "id": "links",
+                "value": [
+                  {
+                    "targetBlank": true,
+                    "title": "Checker Documentation",
+                    "url": "/public/cmos-doc-redirect.html#/cmos/cmos/0.1/checkers.html#${__data.fields.ID}"
```

Comment:
What happens in OSS mode? Should we see this fire - presumably not?

Reply:
The table will be broken and empty anyway (that's a separate issue), so this shouldn't ever be visible